Fri Sep  8 20:09:14 CEST 2006  AlexExtreme <alex@alex-smith.me.uk>
  * netconfig: forgot to change docs accordingly
diff -rN -u old-frugalwareutils/netconfig/doc/netconfig.5 new-frugalwareutils/netconfig/doc/netconfig.5
--- old-frugalwareutils/netconfig/doc/netconfig.5	2006-09-08 19:38:07.000000000 +0200
+++ new-frugalwareutils/netconfig/doc/netconfig.5	2006-09-08 19:38:07.000000000 +0200
@@ -100,7 +100,7 @@
 specify here the dhcp hostname, then you'll need "-t 10 -h <hostname>".
 .TP
 .B "dhcpclient = <dhcp client>"
-This specifies the dhcp client to use. It can be either dhcpcd or dhclient (default).
+This specifies the dhcp client to use. It can be either dhcpcd (default) or dhclient.
 .SH AUTHORS
 Written by Miklos Vajna.
 .SH "REPORTING BUGS"

